spec/spec_helper.rb

```diff
@@ -0,0 +1,104 @@
+require 'simplecov'
+SimpleCov.start do
+  add_filter '/spec/'
+end
+
+# This file was generated by the `rspec --init` command. Conventionally, all
+# specs live under a `spec` directory, which RSpec adds to the `$LOAD_PATH`.
+# The generated `.rspec` file contains `--require spec_helper` which will cause
+# this file to always be loaded, without a need to explicitly require it in any
+# files.
+#
+# Given that it is always loaded, you are encouraged to keep this file as
+# light-weight as possible. Requiring heavyweight dependencies from this file
+# will add to the boot time of your test suite on EVERY test run, even for an
+# individual file that may not need all of that loaded. Instead, consider making
+# a separate helper file that requires the additional dependencies and performs
+# the additional setup, and require it from the spec files that actually need
+# it.
```

Comment:
- [ ] This did not allow me to run the tests. It only worked after I commented it out. Kindly fix this before merging.

![image](https://user-images.githubusercontent.com/61479648/91331625-5af55580-e7a1-11ea-873f-02f22857e40e.png)

Reply:
Hi, I'm using "simplecove" gem for test coverage so these lines were written for that gem. Let me update my README as well .